engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/keyed.scala

```diff
@@ -115,4 +120,21 @@ object keyed {
 
   }
 
+  trait KeyEnricher {
```

Comment:
maybe we should consider a different name for such a functionality so that it wouldn't mix with enrichers that are components, unfortunately I have come up with a better name...

Reply:
Why is it trait? Can't we just add methods to object?

Reply:
good point - changed from trait to object. name will be left the same until we find some better name for that ...